### PR TITLE
Fix of Gradio Upload/Mic thing. Force Dark mode theme + always same URL.

### PR DESCRIPTION
**Pull request by FindMuck**
Fixed Upload/Mic thing
image
+...

Idk do Applio need this cool small-tiny feature? Force Dark mode theme + always (even on reload or url change) same url 127.0.0.1:6969/#🍏(localhost:6969/#🍏).

I can adjust code if just one of two features is needed.

Demonstration (video was done before fixing Upload/Mic thing and demonstrating only "Force Dark mode theme + always same URL" feature):

https://github.com/IAHispano/Applio/assets/15386194/8469a9e2-614a-4ad0-8ff0-78995b054509

### CODE REVIEW

**Comment by blaise-tk** [comment]
I think that mixing javascript with python is not the best idea, so I think we will go more for the solution we did in applio v2

File: app.py [hunk 1]
```diff
@@ -16,10 +16,12 @@
 from tabs.download.download import download_tab
 from tabs.tts.tts import tts_tab
 from assets.discord_presence import rich_presence
+from assets.applio_javascript import copy_javascript
 
 rich_presence()
+copy_javascript()
 
-with gr.Blocks(theme="ParityError/Interstellar", title="Applio") as Applio:
+with gr.Blocks(theme="ParityError/Interstellar", title="Applio", head='<script src="./assets/Applio_script.js"></script>') as Applio:
```
Comment (FindMuck) [inline app.py:24]: changed """ to '

Comment (FindMuck) [inline app.py:24]: change `"""` to `'`

**Comment by FindMuck** [comment]
@blaise-tk 
> I think we will go more for the solution we did in applio v2

What was that solution? (I'm new to Applio, sorry)

> I think that mixing javascript with python is not the best idea

Is it really "mixing"? It's more about implementing JavaScript to the header (`<head>`) of Gradio blocks (`gr.Blocks`). [Here](https://www.gradio.app/docs/blocks#initialization) and [here](https://www.gradio.app/guides/custom-CSS-and-JS#adding-custom-java-script-to-your-demo) some more info about it. Please correct me if I am wrong.

File: app.py [hunk 2]
```diff
@@ -16,10 +16,12 @@
 from tabs.download.download import download_tab
 from tabs.tts.tts import tts_tab
 from assets.discord_presence import rich_presence
+from assets.applio_javascript import copy_javascript
 
 rich_presence()
+copy_javascript()
 
-with gr.Blocks(theme="ParityError/Interstellar", title="Applio") as Applio:
+with gr.Blocks(theme="ParityError/Interstellar", title="Applio", head="<script src='./assets/Applio_script.js'></script>") as Applio:
```
Comment (FindMuck) [inline app.py:24]: Better solution with `"` instead of `"""`

**Comment by aitronssesin** [comment]
We finally fixed the "Upload/Mic thing" without using any javascript, there was an unnecesary gr.Row.

**Pull request closed without merging**